settings.gradle

```diff
@@ -1,6 +1 @@
-pluginManagement {
-    repositories {
-        gradlePluginPortal()
-    }
-}
-rootProject.name = 'court-list-service'
+rootProject.name = 'court-list-service'
```

Comment:
Are we calling the application court-list-service instead of prepare-a-case-service?